diff --git a/lib/index.js b/lib/index.js
index 919bb233027c2fb894a7a7bbe08d487966d2d70e..ca737a573922bdee5e9df02b1f06ff9da7a5744a 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -38,7 +38,7 @@ const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_typografi_1 = __importDefault(require("nav-frontend-typografi"));
-require("nav-frontend-etiketter-style");
+require("nav-frontend-etiketter-style/src/index.less");
 const cls = (type, mini, className) => (0, classnames_1.default)("etikett", className, {
     "etikett--advarsel": type === "advarsel",
     "etikett--fokus": type === "fokus",