diff --git a/lib/Lenkepanel-base.js b/lib/Lenkepanel-base.js
index 8a7a74d30b07209e97316ca779da9fccf6525f0c..5d1562e7c6c477032e017fb0b59b274f75a56d52 100644
--- a/lib/Lenkepanel-base.js
+++ b/lib/Lenkepanel-base.js
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
-require("nav-frontend-lenkepanel-style");
+require("nav-frontend-lenkepanel-style/src/index.less");
 const cls = (className, border) => (0, classnames_1.default)("lenkepanel", className, {
     "lenkepanel--border": border,
 });
diff --git a/lib/index.js b/lib/index.js
index b0e8a3fd23e2c6888df8fad0ec2664b508c6002c..6ea3ddcca994dfa3ea7bd39a54d14ed1532436be 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -37,7 +37,7 @@ exports.LenkepanelBase = void 0;
 const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const nav_frontend_typografi_1 = __importDefault(require("nav-frontend-typografi"));
-require("nav-frontend-lenkepanel-style");
+require("nav-frontend-lenkepanel-style/src/index.less");
 const Lenkepanel_base_1 = __importDefault(require("./Lenkepanel-base"));
 class Lenkepanel extends React.PureComponent {
     render() {