diff --git a/lib/column.js b/lib/column.js
index 97c6411f4fcb1fd23e768ee52a556f7615689a95..ddeda541eb05ca9df3f63e3e80ada166d92a594a 100644
--- a/lib/column.js
+++ b/lib/column.js
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
-require("nav-frontend-grid-style");
+require("nav-frontend-grid-style/src/index.less");
 const cls = (className, xs, sm, md, lg) => (0, classnames_1.default)("col", className, {
     [`col-xs-${xs}`]: !!xs,
     [`col-sm-${sm}`]: !!sm,
diff --git a/lib/container.js b/lib/container.js
index 4a0e8d012c84bf2ceb308e96900e534dc404400b..5bc808fcccc4adbfea1b3e1a3ec662400c8afda2 100644
--- a/lib/container.js
+++ b/lib/container.js
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
-require("nav-frontend-grid-style");
+require("nav-frontend-grid-style/src/index.less");
 const cls = (fluid, className) => (0, classnames_1.default)(className, {
     container: fluid === false,
     "container-fluid": fluid === true,
diff --git a/lib/row.js b/lib/row.js
index 4cdb3fad2ef60e432b16a716e6847412ff89965c..7785d68c4b140cccec855387e242992eb4ec5191 100644
--- a/lib/row.js
+++ b/lib/row.js
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
-require("nav-frontend-grid-style");
+require("nav-frontend-grid-style/src/index.less");
 const cls = (className) => (0, classnames_1.default)("row", className, {});
 class Row extends React.Component {
     render() {