diff --git a/lib/lukknapp.js b/lib/lukknapp.js
index e989cf789260d9f4c6f16af70b670a277062c9c7..cad23c063f15d6d40a35e03b5efdc006916c2948 100644
--- a/lib/lukknapp.js
+++ b/lib/lukknapp.js
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
-require("nav-frontend-lukknapp-style");
+require("nav-frontend-lukknapp-style/dist/main.css");
 const cls = (bla, hvit, hjorne, className) => (0, classnames_1.default)("lukknapp", {
     "lukknapp--hvit": hvit,
     "lukknapp--bla": bla,