diff --git a/lib/ekspanderbartpanel-base.js b/lib/ekspanderbartpanel-base.js
index ad8d4b3cb9f6ad3e2772ffd3fb2ab9c9ddb70b5d..6a713b47b831febe70abb9beb2c4afc14e2198ea 100644
--- a/lib/ekspanderbartpanel-base.js
+++ b/lib/ekspanderbartpanel-base.js
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const react_collapse_1 = require("react-collapse");
-require("nav-frontend-ekspanderbartpanel-style");
+require("nav-frontend-ekspanderbartpanel-style/src/index.less");
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const cls = (props) => (0, classnames_1.default)("ekspanderbartPanel", props.className, {
     "ekspanderbartPanel--lukket": !props.apen,
diff --git a/lib/index.js b/lib/index.js
index f75e0a515a6e43cf67284567f3835984877db34d..b3588e5b70cbe57b5de0e87324a11924697bd59e 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -24,7 +24,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.EkspanderbartpanelBase = void 0;
 const React = __importStar(require("react"));
-require("nav-frontend-ekspanderbartpanel-style");
+require("nav-frontend-ekspanderbartpanel-style/src/index.less");
 const ekspanderbartpanel_base_1 = __importDefault(require("./ekspanderbartpanel-base"));
 class Ekspanderbartpanel extends React.Component {
     constructor(props) {